src/Collections-Strings-Tests/StringTest.class.st

```diff
@@ -1461,6 +1461,14 @@ StringTest >> testIsWideString [
 	self deny: 'éàôüÖ' isWideString
 ]
 
+{ #category : #tests }
+StringTest >> testKeywords [
+
+	self assert: (#foo: keywords) equals: #('foo:').
+	self assert: (#foo:bar: keywords) equals: #('foo:' 'bar:').
```

Comment:
The test should also show the case for unary and binary messages

Reply:
This is the test for the method that should only accept keyword-based selectors, so there shouldn't be any need to test